src/Engine/Army.java

```diff
@@ -282,15 +306,36 @@ public GameEventQueue receiveMassDamageEvent(Commander attacker, Map<Unit, Integ
   {
     if( this == attacker.army )
       return null; // Punching yourself shouldn't make you angry
+    if( TagMode.AWBW == gameRules.tagMode
+        && getAbilityText().length() > 0 )
+      return null; // If any power is active, deny charge in AWBW mode
```

Comment:
Could we put this check in one place instead of two? Maybe in `calculateCombatCharge`? I guess this way prevents superfluous calculations though. Hm.

Reply:
I remember thinking about that and not coming up with anything I liked.